libs/core/src/lib/menu/menu.component.ts

```diff
@@ -170,6 +199,20 @@ export class MenuComponent
         });
     }
 
+    /** Register addon */
+    registerAddon(addon: unknown): void {
+        if (!this._addons.includes(addon)) {
```

Comment:
Why not use `Set`?

Reply:
Then I'd have to expose addons too, which I did not want. We just add and remove, that's all what's needed

Reply:
I mean set instead of array

Reply:
Oh, then my bad